### PR TITLE
[SmartSwitch] Extend implementation of the DPU chassis daemon.

### PR DESCRIPTION
**Pull request by oleksandrivantsiv**

### CODE REVIEW

File: sonic-chassisd/scripts/chassisd [hunk 1]
```diff
@@ -790,21 +790,74 @@ class ChassisdDaemon(daemon_base.DaemonBase):
...
+            swsscommon.SubscriberStateTable(self.app_db, 'PORT_TABLE'),
+            swsscommon.SubscriberStateTable(self.state_db, 'SYSTEM_READY')
+        ]
+
+        for s in selectable:
+            sel.addSelectable(s)
+
+        try:
+            while True:
+                (state, c) = sel.select(SELECT_TIMEOUT)
```
Comment (vivekrnv) [inline sonic-chassisd/scripts/chassisd:815]: Where is SELECT_TIMEOUT defined?

File: sonic-chassisd/scripts/chassisd [hunk 2]
```diff
@@ -790,21 +790,74 @@ class ChassisdDaemon(daemon_base.DaemonBase):
...
+        selectable = [
+            swsscommon.SubscriberStateTable(self.app_db, 'PORT_TABLE'),
+            swsscommon.SubscriberStateTable(self.state_db, 'SYSTEM_READY')
+        ]
+
+        for s in selectable:
+            sel.addSelectable(s)
+
+        try:
+            while True:
```
Comment (vivekrnv) [inline sonic-chassisd/scripts/chassisd:814]: Looks like this subprocess might not handle sigterm gracefully. self.stop should be leveraged here i think

Comment (oleksandrivantsiv) [reply sonic-chassisd/scripts/chassisd:814]: The worker is stateless, the is nothing to handle during the shutdown

File: sonic-chassisd/scripts/chassisd (hunk 1 above)
Comment (oleksandrivantsiv) [reply sonic-chassisd/scripts/chassisd:815]: Line 85 in this file:
SELECT_TIMEOUT = 1000

**Review by vivekrnv — APPROVED** [review]

**Review by dgsudharsan — APPROVED** [review]

**Pull request closed without merging**